kernel/Makefile

```diff
@@ -0,0 +1,196 @@
+# OS X kext makefile
+VERSION:=$(shell git describe --tags HEAD --always)
+VERSION_DEFINE:=-DVERSION="\"$(VERSION)\""
+
+CC:=clang
+CXX:=clang++
+
+# Directories
+INCLUDE_DIR:=include/
+BUILD_DIR:=build/
+SOURCE_DIR:=src/
+KEXT:=osquery.kext
+CONTENTS_DIR:=$(BUILD_DIR)$(KEXT)/Contents/
+OUTPUT_DIR:=$(CONTENTS_DIR)MacOS/
+RUNDIR:=/tmp/
+USER_DIR:=user/
+
+INCLUDES:=$(wildcard $(INCLUDE_DIR)*.h)
+INCLUDE:=-I$(INCLUDE_DIR) -I/System/Library/Frameworks/Kernel.framework/Headers
```

Comment:
You could also have a make target symlink this to `/System/Library/Frameworks/Kernel.framework/Headers`